### PR TITLE
Use default userns field if set in containers.conf

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1995900

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: cmd/podman/common/specgen.go [hunk 1]
```diff
@@ -248,6 +249,10 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 		}
 	}
 
+	if !registry.IsRemote() && c.UserNS == "" {
```
Comment (mheon) [inline cmd/podman/common/specgen.go:252]: What if the user explicitly set `--userns=""` - is there any way to do this higher up in the stack, where we know whether the flag was changed?

**Comment by kriansa** [comment]
Hi @rhatdan,

Is this good to merge?

**Comment by rhatdan** [comment]
@kriansa If it passes tests, yes.

**Comment by TomSweeneyRedHat** [comment]
Changes look fine, but all kinds of red test unhappiness @rhatdan

**Pull request closed without merging**